service/src/main/resources/db/changesets/20230711.yaml

```diff
@@ -0,0 +1,25 @@
+# Add pipeline_inputs table
+databaseChangeLog:
+  - changeSet:
+      id: add pipeline_inputs table
+      author: js
+      changes:
+        - createTable:
+            tableName: pipeline_inputs
+            remarks: |
+              There is one row in the pipeline_inputs table per job submitted to the service.
+            columns:
+              - column:
+                  name: job_id
+                  type: UUID
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    foreignKeyName: pipeline_inputs_job_id_FK
```

Comment:
I wasn't sure if it mad more sense for the FK to be `jobs.job_id` or `jobs.id` and i went with `jobs.job_id`.  Curious on your thoughts on this

Reply:
It's not a huge deal, but I'd normally just stuck with jobs.id.  Then, in any other tables that use it as a fk, I'd make THOSE other_table.job_id.  But again, it ultimately doesn't matter much.